dataland-frontend/src/components/resources/dataTable/cells/DataPointDisplayComponent.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <span>{{ content.displayValue.label }}</span>
+  <DocumentLink
+    label=""
+    :suffix="formattedPageNumber"
+    :download-name="content.displayValue.fileName"
+    :file-reference="content.displayValue.fileReference"
+    show-icon
+  />
+</template>
+
+<script lang="ts">
+import { defineComponent } from "vue";
+import {
+  type MLDTDisplayComponentName,
+  type MLDTDisplayObject,
+} from "@/components/resources/dataTable/MultiLayerDataTableCellDisplayer";
+import DocumentLink from "@/components/resources/frameworkDataSearch/DocumentLink.vue";
+
+export default defineComponent({
```

Comment:
When hovering over the download link, the file name of the file that would be downloaded should be shown. This helps the user to not 20 times download the same file.

Reply:
This was easily implemented. However it will probably be removed away by DALA-2156 introducing modals instead of download links at this place.

Reply:
yes right